package.json

```diff
@@ -24,33 +24,34 @@
     "@expo-google-fonts/roboto": "0.2.3",
     "@expo-google-fonts/rubik": "0.2.3",
     "@expo/cli": "^0.10.11",
-    "@ory/client": "1.1.46",
+    "@ory/client": "1.2.0",
     "@ory/themes": "0.0.21",
     "@react-native-async-storage/async-storage": "1.18.2",
     "@react-native-community/async-storage": "^1.12.1",
     "@react-native-community/masked-view": "^0.1.11",
     "@react-navigation/native": "~6.1.7",
     "@react-navigation/stack": "~6.3.17",
+    "@redtea/format-axios-error": "2.1.1",
     "@sentry/integrations": "7.64.0",
     "@sentry/react-native": "5.5.0",
     "@types/node": "~18.17.5",
     "axios": "~1.4.0",
     "expo": "^49.0.0",
     "expo-application": "~5.3.0",
     "expo-auth-session": "~5.0.2",
-    "expo-cli": "~6.0.6",
+    "expo-cli": "6.0.8",
     "expo-constants": "~14.4.2",
     "expo-device": "~5.4.0",
     "expo-linking": "~5.0.2",
     "expo-random": "~13.2.0",
     "expo-secure-store": "~12.3.1",
     "expo-status-bar": "~1.6.0",
-    "expo-updates": "~0.18.11",
+    "expo-updates": "~0.18.12",
```

Comment:
Could we also remove the tilda `~` and caret `^` symbols from the package versions? Makes it easier to prevent accidental breaking changes in the app.